src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml

```diff
@@ -25,40 +25,76 @@
-            <h2 class="govuk-heading-m">@Model.Opportunities.Count() @(Model.Opportunities.Count() == 1 ? "opportunity" : "opportunities")</h2>
-            <div class="das-search-results">
-                <ol class="das-search-results__list">
-                    @foreach (var opportunity in Model.Opportunities)
-                    {
-                    <li class="das-search-results__list-item">
-                        <h2 class="govuk-heading-m das-search-results__heading govuk-!-margin-bottom-0">
-                            <a href="@Url.Action("Detail", new { EncodedPledgeId = opportunity.ReferenceNumber })" class="das-search-results__link">@opportunity.EmployerName (@opportunity.ReferenceNumber)</a>
-                        </h2>
-                        <p class="govuk-hint">up to @opportunity.DisplayAmount available in @Model.TaxYear</p>
-                        <ul class="govuk-list govuk-!-margin-bottom-0">
-                            <li>
-                                <strong>Locations: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Locations</span>
-                            </li>
-                            <li>
-                                <strong>Sectors: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Sectors</span>
-                            </li>
-                            <li>
-                                <strong>Type of role: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.JobRoles</span>
-                            </li>
-                            <li>
-                                <strong>Levels: </strong>
-                                <span class="das-!-color-dark-grey">@opportunity.Levels</span>
-                            </li>
-                        </ul>
-                    </li>
-                    }
-                </ol>
-            </div>
         </div>
     </div>
+    <div class="govuk-grid-row">
+        <form method="post">
+            <div class="govuk-grid-column-one-third">
+                <h2 class="govuk-heading-m">Apply filter</h2>
+                <details class="govuk-details" data-module="govuk-details">
+                    <summary class="govuk-details__summary">
+                        Your business sector
+                    </summary>
+                    <div class="govuk-details__text">
+                        <div class="govuk-form-group">
+                            <fieldset class="govuk-fieldset">
+                                <legend class="govuk-visually-hidden">Select one or more business sectors to filter by  </legend>
+                                <checkbox-list asp-for="Sectors" source="@Model.Sectors" css-class="govuk-checkboxes govuk-checkboxes--small"></checkbox-list>
```

Comment:
It might be worth checking with UX or BA, but I'd have thought that if I had applied a filter then the appropriate filter would be open when the page re-renders. Could be wrong tho.